docs/en/README.md

```diff
@@ -7,7 +7,7 @@
 - [Installation](installation.md)
 - [What is Vuex?](intro.md)
 - [Getting Started](getting-started.md)
-- Core Concepts
+- [Core Concepts](state.md)
```

Comment:
Oh, sorry. I had intended it to have core-concepts.md link only. Somehow missed it.